package.json

```diff
@@ -34,7 +34,8 @@
     "babel-plugin-jsx-event-modifiers": "^2.0.2",
     "babel-plugin-jsx-v-model": "^2.0.1",
     "babel-plugin-syntax-jsx": "^6.18.0",
-    "babel-plugin-transform-vue-jsx": "^3.5.0"
+    "babel-plugin-transform-vue-jsx": "^3.5.0",
+    "babel-plugin-vue-jsx-sync": "^0.0.4"
```

Comment:
seems it's still too young and lacks of tests?

Reply:
@egoist yeah,  it's young 😂 . I use it in my own project, and it works currently.

Reply:
We should give it some time to mature first 😄

Reply:
@egoist OK, Tnx for your time.

Reply:
@egoist, maybe we can add `sync` to [event-modifiers](https://github.com/nickmessing/babel-plugin-jsx-event-modifiers), what do you think?

Reply:
@njleonzhang, if you could make a PR to [event-modifiers](https://github.com/nickmessing/babel-plugin-jsx-event-modifiers), I'd be very grateful

Reply:
@nickmessing, OK, I will try to make one after spring festival